### DISCUSSION TITLE
My site is not working properly

### DISCUSSION THREAD

**Discussion by MaiconDz** [community/community#32051] [Other Feature Feedback, Questions, & Ideas]
I have uploaded two scripts into my repository, one in HTML and one in CSS, the HTML is the index for my sales page, and the CSS is for the style of the page. While in pc browsers, the site works as intended, in any cellphone the images and texts just dont stay as i programmed. When i looked in the font code of the page, i sew that the CSS code that i uploaded had not yet been updated, and was still the old version. But the HTML code did update, and i did upload them in the same time... So i deleted everything in the repository and uploaded again about 1 hour ago... In the github menu i can access my site and it works fine, but if i try to access it with the normal URL, it shows me the error 404 as if i didnt upload any index into the repository, can anyone help me?

**Comment by radar-blip** [comment]
Seems to be working just fine now.

**Reply by MaiconDz** [reply]
Yeah i found out what was the error.
Aparently my domain was removed for no reason, so i just had to put that again and worked.
I dunno why my domain was removed, the other day was working fine, but its okay now, tks

**Reply by 3kh0** [reply]
@MaiconDz Please mark this as the answer to help others
